mmdet/integration/nncf/compression_hooks.py

```diff
@@ -5,7 +5,7 @@
 from mmcv.runner.dist_utils import master_only
 
 
-@HOOKS.register_module()
+@HOOKS.register_module(force=True)
```

Comment:
Please add comment with reason to add force argument, something like "force=true added to work with mmdet and mmseg in one environment"

Reply:
do you mean that some comment should be on modified codes, right?

Reply:
Yes 
```suggestion
# Argument 'force' used to work with mmdetection and mmsegmentation repositories in one environment. 
@HOOKS.register_module(force=True)
```